mne/decoding/receptive_field.py

```diff
@@ -31,11 +31,13 @@ class ReceptiveField(BaseEstimator):
     coef_ : array, shape (n_features, n_delays)
         The coefficients from the model fit, reshaped for easy visualization.
         If you want the raw (2d) coefficients, use `mne.decoding.get_coefs`
-        on `self.model`.
-    mask_pred : array, dtype bool, shape (n_times,)
+        on `self.estimator_`.
+    mask_pred : array of bool, shape (n_times,)
         A mask with True values corresponding to datapoints that were used
         in generating model predictions. Creating time delays necessitates that
-        datapoints at the beginning / end of the input are not used.
+        datapoints at edges of the input are not used because of missing
+        datapoints at the beginning or end of the data.
```

Comment:
how about "datapoints at edges of the input do not have matching time-lagged datapoints for all required lags, and are thus removed in model fitting"?